webpack/configUtils.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
I don't think we need this file, IMO it's an unnecessary complexity.

Reply:
I also agree, I fill fix it as soon as I find a way to share same matchers between files

Reply:
I think adding this helper function adds too much complexity.
I would either leave them hardcoded or just add constants for it.